interfaces/builtin/network_control.go

```diff
@@ -188,6 +189,9 @@ capability setuid,
 /run/network/ifup-* rw,
 /run/network/ifdown-* rw,
 
+# iptables
```

Comment:
This is already covered by the `firewall-control` interface - I don't think it is appropriate for `network-control` to gain this as it seems too much like scope creep compared to what it currently covers.